src/common/deployPipelineCache.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Global, Messages, TTLConfig } from '@salesforce/core';
+import { Duration } from '@salesforce/kit';
+import { JsonMap } from '@salesforce/ts-types';
+
+Messages.importMessagesDirectory(__dirname);
+const cacheMessages = Messages.load('@salesforce/plugin-devops-center', 'cache', ['error.NoRecentAorId']);
+
+// For the moment we will just store the timestamp
+export type AORData = Record<string, never>;
```

Comment:
```suggestion
export type AsyncOperationData = Record<string, never>;
```

This might be a bit more readable.  Just a suggestion.

Reply:
Ok!